MODULE.bazel

```diff
@@ -23,14 +23,19 @@ use_repo(node, "nodejs_linux_s390x")
 use_repo(node, "nodejs_linux_ppc64le")
 use_repo(node, "nodejs_windows_amd64")
 
-pnpm = use_extension("@aspect_rules_js//npm:extensions.bzl", "pnpm")
+####### Dev dependencies ########
+
+pnpm = use_extension(
+    "@aspect_rules_js//npm:extensions.bzl",
+    "pnpm",
+    dev_dependency = True,
```

Comment:
FYI @kormide I think it was a mistake that we installed a fixed pnpm version for everyone, as this was not a devdep, e2e/js_run_devserver was unable to select a different pnpm version.
Sound right?

Reply:
It was introduced in https://github.com/aspect-build/rules_js/pull/763

Reply:
I agree that it was a mistake. I think at the time I was trying to make the module extension do what the workspace rules does, but those are different idioms.

Reply:
oh, it's not this simple. rules_js references pnpm in load statements, so it does have to declare the dependency.